examples/sig_test.go

```diff
@@ -89,30 +89,30 @@ func SchnorrVerify(suite Suite, message []byte, publicKey kyber.Point,
 // Example of using Schnorr
 func Example_schnorr() {
 	// Crypto setup
-	group := edwards25519.NewAES128SHA256Ed25519()
-	rand := group.Cipher([]byte("example"))
+	suite := edwards25519.NewAES128SHA256Ed25519()
```

Comment:
Any ideas of what makes sense here? The `NewAES...` has more definitions than we need. Is that OK?

Reply:
I don't understand the question.

Reply:
With the new definition of `Suite`, we tell the outside world what we need. In this case it's: `Group`, `Encoding`, and `XOFFactory`. But the `New...` gives also the hash and the symmetric encryption. But that's probably not a problem, is it?